app/controllers/casa_cases_controller.rb

```diff
@@ -40,8 +40,14 @@ def edit
   end
 
   def create
+    create_params = if casa_case_params[:empty_court_date] == "1"
```

Comment:
How about creating a private method here that describes the condition, like:

```
  def create
    create_params = if court_date_unknown?
    ...
    end
  end

...


  private

  def court_date_unknown?
    casa_case_params[:empty_court_date] == "1"
  end
```

Then it is a little bit easier to read the code like a sentence and understand what is happening.

Reply:
Agreed! I refactored it, creating a casa_case_create_params, so the create method is cleaner now.